libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -211,6 +229,77 @@ void HarmonicNotchFilter<T>::expand_filter_count(uint16_t total_notches)
     delete[] _old_filters;
 }
 
+/*
+  set the center frequency of a single notch
```

Comment:
needs a better comment since there are more arguments than just the center frequency for a notch

Reply:
done